net/netfilter/nf_tables_api.c

```diff
@@ -8391,18 +8413,25 @@ static void nf_tables_cleanup(struct net *net)
 static int nf_tables_abort(struct net *net, struct sk_buff *skb,
 			   enum nfnl_abort_action action)
 {
-	int ret = __nf_tables_abort(net, action);
+	struct nftables_pernet *nft_net = nft_pernet(net);
+	unsigned int gc_seq;
+	int ret;
 
+	gc_seq = nft_gc_seq_begin(nft_net);
+        ret = __nf_tables_abort(net, action);
+        nft_gc_seq_end(nft_net, gc_seq);
 	mutex_unlock(&net->nft_commit_mutex);
 
 	return ret;
 }
 
 static bool nf_tables_valid_genid(struct net *net, u32 genid)
 {
+	struct nftables_pernet *nft_net = nft_pernet(net);
 	bool genid_ok;
 
 	mutex_lock(&net->nft_commit_mutex);
+	nft_net->tstamp = get_jiffies_64();
```

Comment:
This is teachincally not in `4.18.0-534`
https://github.com/ctrliq/kernel-src-tree/blame/centos_kernel-4.18.0-534.el8/include/net/netfilter/nf_tables.h#L1535

The do however bring in the `tstamp` variable later as in `4.18.0-553.7.1` they do introduce
https://github.com/torvalds/linux/commit/7395dfacfff65e9938ac0889dafa1ab01e987d15#diff-233a9dea2c513f5d208bba55e0f8ef4e8a29f5c584d30b5815cfa6c2cf5361ecR10626
```
* Thu Jun 06 2024 Denys Vlasenko <dvlasenk@redhat.com> [4.18.0-553.7.1.el8_10]
- netfilter: nf_tables: use timestamp to check for set element timeout (Phil Sutter) [RHEL-38023] {CVE-2024-27397}

* Wed May 29 2024 Denys Vlasenko <dvlasenk@redhat.com> [4.18.0-553.6.1.el8_10]
```

Reply:
this is gone in the next push - was a big cock up on my part.  It's not in 534 so I've removed it as unnecessary.